src/commands/console.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const AppConfig = require('../models/app_configuration.js');
+const Domain = require('../models/domain.js');
+const handleCommandStream = require('../command-stream-handler.js');
+const Logger = require('../logger.js');
+const OpenBrowser = require('../open-browser.js');
+
+function console (api, params) {
+  const { alias } = params.options;
+  const s_open = AppConfig.getAppData(alias)
+    .flatMapLatest(({ app_id, org_id }) => {
+      Logger.println('Opening the console in your browser');
+      return OpenBrowser.openPage('https://console.clever-cloud.com/users/me/applications/' + app_id);
```

Comment:
I haven't tested but I think it won't work for apps in organizations.
We miss a test on the `org_id` to build the right URL in any case.

I also think we should use template strings for this:

```js
const appUrl = `https://console.clever-cloud.com/users/me/applications/${app_id}`;
```